src/frontend/komponenter/ManuellJournalføring/DokumentVelger.tsx

```diff
@@ -0,0 +1,132 @@
+import { IDokumentInfo, IJournalpost } from '@navikt/familie-typer';
+import Panel from 'nav-frontend-paneler';
+import React, { useState } from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import { DokumentIkon } from '../../ikoner/DokumentIkon';
+import { useManuellJournalføring } from '../../context/ManuellJournalføringContext';
+
+const DokumentPanel = styled(Panel)`
+    margin-top: 20px;
+    width: 27rem;
+    height: 100%;
+    border: ${props => `${props.theme.borderWidth} solid ${props.theme.borderColor}`};
+    &:hover {
+        border-color: ${props => `${props.theme.hoverBorderColor}`};
+    }
+`;
+
+const DokumentInfoStripeContainer = styled.div`
+    display: flex;
+    justify-content: left;
+    align-items: flex-start;
+    width: 100%;
+    height: 100%;
+`;
+
+interface IDokumentInfoProps {
+    dokument: IDokumentInfo;
+    journalpost: IJournalpost;
+}
+
+interface IDokumentInfoStripeProps {
+    dokument: IDokumentInfo;
+    journalpost: IJournalpost;
+    valgt: boolean;
+    utvidet: boolean;
+    onClick: () => void;
+}
+
+interface IDokumentVelgerProps {
+    dokument: IDokumentInfo;
+    journalpost: IJournalpost;
+    valgt: boolean;
+}
+
+const DokumentTittelDiv = styled.td`
```

Comment:
I find it a bit confusing that this i named Div when it is actually a data cell semantically 👀

Reply:
Fixed!